playbook.yml

```diff
@@ -17,63 +17,84 @@
           - rsync
           - wget
 
-    - name: Install membres page code
-      git:
-        repo: https://github.com/epicerieledetour/ledetour-membres.git
-        dest: /root/ledetour-membres
-
-    - name: Update ledetour-membres website
-      synchronize:
-        src: /root/ledetour-membres/src/
-        dest: /srv/www/ledetour-membres
-        delete: yes
-        recursive: yes
-      delegate_to: "{{ inventory_hostname }}"
+    - name: Download charlesfleche keys
```

Comment:
Instead of `get_url` and `shell` I'd have used the module [authorized_key](https://docs.ansible.com/ansible/2.3/authorized_key_module.html#examples).